AUTHENTICATION.md

```diff
@@ -5,19 +5,93 @@ The recommended way to authenticate to the Google Cloud PHP library is to use
-Specific instructions and environment variables for each individual service are linked from the README documents listed below for each service.
+## Quickstart
+
+Here is a quick an opinionated quickstart guide that serves as a means to get started as quick as possible. There are different ways to authenticate
+a client that we will go over this guide, but we wanted to offer a quick way to configure the authentication. We will go over the steps of what needs to be done in
+order to authenticate a client first and then we will explain said steps in detail for a quickstart.
+
+### For local development:
+* Install the Google Cloud CLI.
+* Authenticate with `gcloud` to generate the credentials file.
+* Instantiate a client.
+
+### Installing the Google Cloud CLI
+In order to generate our needed credentials file we need to authenticate to gcloud first. Installation is handled differently depending on your platform.
+Here is a link to help you setup the Google Cloud CLI:
+
+https://cloud.google.com/sdk/docs/install
+
+### Authenticate via the `gcloud` command
+Once the Google Cloud CLI tools are installed it is required that we authenticate via the `gcloud`:
+```shell
+$ gcloud init
+$ gcloud auth application-default login
+```
 
-## Project and Credential Lookup
+This will create a local file in your system that the authentication library for our client will read in order to make requests to the apis with those credentials.
+This file is located in different place depending on your system.
 
-The Google Cloud PHP library provides several mechanisms to configure your system without 
-providing **Project ID** and **Service Account Credentials** directly in code.
+Windows:
+```
+%APPDATA%\gcloud\application_default_credentials.json
+```
+
+Linux and MacOS:
+```
+$HOME/.config/gcloud/application_default_credentials.json
+```
+
+### Instantiating the client
+Once we have the `application_default_credentials.json` that we created on the previous step now we can instantiate a client which internally using the Google Auth library
+will take that file and use it to authenticate your requests:
+```php
+use Google\Cloud\Translate\V3\Client\TranslationServiceClient;
+use Google\Cloud\Translate\V3\TranslateTextRequest;
 
-**Project ID** is discovered in the following order:
+// Instantiating the client gathers the credentials from the `application_default_credentials.json`
+$client = new TranslationServiceClient();
 
-1. Specify project ID in code
-2. Discover project ID in environment variables 
-3. Discover GCE project ID
+$request = new TranslateTextRequest();
+$request->setParent('<YOUR_PROJECT_ID>');
+$request->setTargetLanguageCode('en-US');
+$request->setContents('こんにちは');
+
+// The request will contain the authentication token based on the default credentials file
+$response = $client->translateText($request);
+
+
+var_dump($response->getTranslations());
```

Comment:
This needs to grab the first element, since it's a repeated field
```suggestion
$translations = $response->getTranslations();
var_dump($translations[0]);
```

Or we could add a loop

```suggestion
foreach($response->getTranslations() as $translation) {
    var_dump($translation);
}
```